src/core/View.js

```diff
@@ -457,14 +458,15 @@ export default class View extends Component {
           JSON.stringify(previous.cameraPosition))
     ) {
       const camera = this.renderer.getActiveCamera();
+      if (previous) {
```

Comment:
Do you mean to check `previous.cameraPosition` here?

Reply:
Ah, I see now. Manual triggering of the resetCamera call with just JSX is not elegant (i.e. look at the triggerRender prop). What if we added a new prop (i.e. `autoResetCamera`) that will automatically reset the camera after the camera is updated? @jourdain 

Additionally, we should have separate checks on cameraPosition, cameraViewUp, and cameraFocalPoint. Something like the following:

```
let cameraUpdated = false;
if (camera{Position,ViewUp,FocalPoint} changed) {
  cameraUpdated = true;
  camera.set{Position,ViewUp,FocalPoint}(camera{Position,ViewUp,FocalPoint});
}

if (cameraUpdated) {
  if (autoResetCamera) this.resetCamera()
  else this.renderWindow.render()
}
```

Reply:
Then is there a way to resetCamera without resetting focal point? Everytime resetCamera is called, the focalpoint is reset too, so no point of setting it.

Reply:
You are correct that resetCamera() will change the focalPoint. I was thinking about the case where if we choose to have an extra `autoResetCamera` prop. If it's false, then the user can choose to freely set the focal point as they see fit. But in thinking about what you need, I presume you want the effect of resetCamera, but with a custom focal point, correct?

Reply:
I'm not sure the best approach here... Would you prefer to call methods on a ref rather than setting props?
We could add a resetCamera() method with some arg that can keep the focalPoint untouched? We could even have some other param to snap the viewUp or else... 

What do you think? dash rely on props, but in a real react app, it might be easier to just call methods?

Reply:
ya either resetCamera effect with custom focalpoint or resetCamera() method with some arg that can keep the focalPoint untouched would work for me.

Reply:
Let's go with something like `View.resetCamera({ focalPoint })` as an option argument, where you can specify a desired focal point that will be set after the camera is reset.